cg/meta/store/mip.py

```diff
@@ -52,15 +50,13 @@ def parse_sampleinfo(data: dict) -> dict:
         dict: parsed data
     """
 
-    sampleinfo_data = {
+    return {
```

Comment:
You know what I think...

Reply:
Yes but its for another large refactor PR